ivy/functional/ivy/device.py

```diff
@@ -494,6 +495,20 @@ def percent_used_mem_on_dev(
     ret
         The percentage used memory on the device.
 
+    Examples
+    --------
+    >>> x = ivy.percent_used_mem_on_dev(device = "cpu", process_specific = False)
+    >>> print(x)
+    94.036902561555
+
+    >>> x = ivy.percent_used_mem_on_dev(device = "cpu", process_specific = True)
+    >>> print(x)
+    0.7024003467681645
+
+    >>> x = ivy.as_native_dev("gpu:0")
+    >>> y = ivy.percent_used_mem_on_dev(device = x, process_specific = False)
+    >>> print(y)
+    0.7095597456708771
```

Comment:
Thanks for making the changes! Just add a linespace after line 511 and you should be done.